quixstreams/dataframe/dataframe.py

```diff
@@ -126,19 +131,24 @@ class StreamingDataFrame:
 
     def __init__(
         self,
-        topic: Topic,
+        *topics: Topic,
         topic_manager: TopicManager,
-        registry: DataframeRegistry,
+        registry: DataFrameRegistry,
         processing_context: ProcessingContext,
         stream: Optional[Stream] = None,
     ):
+        if not topics:
+            raise ValueError("At least one Topic must be passed")
+
         self._stream: Stream = stream or Stream()
-        self._topic = topic
+        self._topics = set(topics)
```

Comment:
EDIT: Please don't apply this comment right away! I think the only place that requires sorted topic names is `TopicManager.stream_id_from_topics`, I left a comment there as well.

---------------------------------

~~Since `_topics` are immutable, we can sort them once.~~
```suggestion
        self._topics = sorted(set(topics), key=lambda t: t.name)
```

~~Additional reason: ensure deterministic behavior now and in the future. Eg. we need topics for `self._registry.register_stream_id` call that constructs two dicts which will have deterministically ordered keys. It's not a problem now, but we can future-proof it for no cost.~~